src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  @include flex-center;
+}
```

Comment:
According to the code style rules, nested selectors should be placed after all properties. Please move the `@include flex-center;` mixin after the `min-height` property.